classes/classes.py

```diff
@@ -0,0 +1,154 @@
+# Task 1
+
+class Snail:
+    def __init__(self, name = 'noname', species = 'unknown', age = 0, size = 0, status = 'inside'):
+        self.name = name
+        self.species = species
+        self.age = age
+        self.size = size
+        self.status = status
+
+    def age_group(self):
+        if self.age <= 15:
+            print('young')
+        elif self.age <= 60:
+            print('adult')
+        else:
+            print('old')
```

Comment:
Здесь лучше возвращать значения, чтобы как-то использовать потом
```suggestion
            return 'young'
        elif self.age <= 60:
            return 'adult'
        else:
            return 'old'
```